From 6c308ef81cfef32b67d12c4f3ab19cc71c19d41e Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Fri, 29 Jul 2022 08:29:20 +0000
Subject: [PATCH 096/120] WIP: arm64: dts: amlogic: fix cvbs disable on WeTek
 Hub

The original submission in commit d537d289de06 ("ARM64: dts: meson-gxbb:
Add support for WeTek Hub and Play") shows cvbs output as disabled, but
this appears to have been dropped accidentally when moving WeTek boards
to a common dtsi. The Hub does not have cvbs hardware so let's fix and
reinstate the disable.

Fixes fb72c03e0e32 ("ARM64: dts: meson-gxbb-wetek: add a wetek specific dtsi to cleanup hub and play2")
Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxbb-wetek-hub.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek-hub.dts b/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek-hub.dts
index 58733017eda8..23bea3fe9501 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek-hub.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek-hub.dts
@@ -13,6 +13,10 @@ / {
 	compatible = "wetek,hub", "amlogic,meson-gxbb";
 	model = "WeTek Hub";
 
+	cvbs-connector {
+		status = "disabled";
+	};
+
 	sound {
 		compatible = "amlogic,gx-sound-card";
 		model = "WETEK-HUB";
-- 
2.34.1

